.changeset/plenty-tigers-marry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@espressive/cascara": mejor
```

Comment:
```suggestion
"@espressive/cascara": patch
```
This should be a patch change. No functionality is changing for the end user. We are only resolving a bug on our end.

Reply:
:o Soooorry I confused the PR, changing that.